tasks/strongloop_deploy.yml

```diff
@@ -0,0 +1,33 @@
+---
+# tasks file for deploying strongloop code
+- name: Set destination for the code
+  set_fact:
+    repo_path: "{{ item.repo | basename | regex_replace('.git','') }}"
+
+- name: Fetch code
+  git:
+    repo={{ item.repo }}
+    dest=/opt/{{ item.project_name }}/{{ repo_path }}
```

Comment:
use {{ item.name }}.

Validate that this would create the parent folder /opt/{{ item.name }} as well - if not, you may have to add an extra step before.